CONTRIBUTING.md

```diff
@@ -112,6 +132,7 @@ It generally describes the "why" and "what" according to the following pattern:
 * the chosen solution and why it was chosen;
 * the (core) change itself - in the imperative (i.e. "add foo()", not ("added foo()");
 * additional changes needed to keep things working (e.g. "Update callers with the new argument").
+* The JIRA key should ideally be part of every commit. It can be either on its own (PPM-204) or as a link to JIRA (https://jira.prplfoundation.org/browse/PPM-204)
```

Comment:
```suggestion
* The JIRA key should ideally be part of every commit.
  It can be either on its own (PPM-204) or as a link to JIRA (https://jira.prplfoundation.org/browse/PPM-204).
  The latter is preferred since it allows single-click browsing to the issue from the shell.
```
But maybe we should just make the URL mandatory?

Reply:
Actually, I think there's a way to automatically link from github to JIRA. And I know that exists for gitlab. So I don't want to mandate including the full link.